### PR TITLE
Bump `postgres` and `mysql` versions

### PR DESCRIPTION
**Pull request by pat-s**
I am running both a mysql and postgres Gitea instance with the new DB versions.

Also Postgres 9.6 is quite old and has its EOL in 2021-11.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by zeripath** [comment]
Hmm. These are supposed to be the minimum versions we support. Is there any reason why we should be bumping these?

**Comment by silverwind** [comment]
The docs update is probably fine but we probably should continue testing the minimum version. Maybe just add a comment to the lines saying that it's the minimum supported version.

**Comment by techknowlogick** [comment]
In terms of minimum supported version perhaps we could bump it to 10, as once 9.6 is eol it is the next lowest supported version per https://www.postgresql.org/support/versioning/

**Comment by pat-s** [comment]
How about suggesting the most recent stable versions in the docs if we know they work while also testing against a minimum version?
We could suggest the most recent version in the doc and add a comment to it which lists the minimum.

The issue I saw is that people reading the docs might read it as "the devs recommend to use this old version because newer do not work".
I guess there might have been cases where people used an older version on purpose because of the docs.

Instead we could give a range of versions which we have tested against.
This gives the opportunity for new users to potentially start with a recent DB version and others to attach Gitea to their old running instances.

**Comment by silverwind** [comment]
Ideally we would test at least the minimum and newest versions but I'm not sure if we have the CI resources to do so.

**Comment by pat-s** [comment]
Sure, testing against multiple versions is always great but would result in an infinite test matrix at some point.

One argument testing against older versions is that they are known to work and and static - and unless there are major internal changes in Gitea, no issues should arise (but ofc there is no guarantee).

Whereas when doing dynamic testing against the latest versions, one might spot issues early because these will update on the fly during CI.

But I did not wanted to trigger a general discussion about CI testing and a possible matrix expansion.
Given that Gitea is a project which likes to use recent dependency releases, I thought it would make sense to bump the DB versions.
From what I've seen during discussions in the last months, many people use more recent DB versions "on their own risk" and I was wondering if the "risk" should be taken care of :)

**Comment by techknowlogick** [comment]
TBH I'm fine with min & latest, as we in theory already do that w/ mysql. If we need more CI resources then we have very friendly (and amazing) sponsors that we hopefully can ask very nicely, or utilize some of our opencollective funds for it (as additional testing is a very reasonable thing to spend our funds on)

**Review by 6543 — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -319,7 +319,7 @@ trigger:
 services:
   - name: pgsql
     pull: default
-    image: postgres:9.5
+    image: postgres:13
```
Comment (6543) [inline .drone.yml]:
```suggestion
    image: postgres:10
```

**Comment by 6543** [comment]
bumped postres version to **10** to tests ci against

**Pull request merged**